experiments/pyexps/ae/data_decmp.py

```diff
@@ -36,12 +36,20 @@
 with open(out_file, 'r', encoding='utf-8') as f:
     lines = f.readlines()
 
+    start_found = False
+    end_found = False
+    start_time = 0
+    end_time = 0
```

Comment:
```suggestion
    start_time = None
    end_time = None
```
There's a simpler and more Pythonic (I hate this word) way to do this.

Reply:
Not sure whether its a good idea to set a numeric value that is not an optional to None.

Reply:
This is just an experiment script that you can even fully view without scrolling. Imo, introducing type annotations here just clutters up the code. If this was part of the orchestration framework itself I would definitely be on your side though.

Reply:
Agreed